src/Components/Common/components/Page.tsx

```diff
@@ -38,7 +38,7 @@ export default function Page(props: PageProps) {
 
   return (
     <div className={classNames(padding, props.className)} ref={props.ref}>
-      <div className="flex flex-col justify-between gap-2 md:flex-row md:items-center md:gap-6">
+      <div className="flex flex-col justify-between gap-2  md:flex-row md:items-center md:gap-6">
```

Comment:
```suggestion
      <div className="flex flex-col justify-between gap-2 md:flex-row md:items-center md:gap-6">
```
You don't need an extra space!